### DISCUSSION TITLE
CRM support

### DISCUSSION THREAD

**Discussion by cr0hn** [tdesposito/pyZohoAPI#9] [Ideas]
Ey! Thank you for your work!

Do you plan to include the Zoho CRM support?

thx!

**Comment by tdesposito** [comment]
Hi cr0hn! No, I have no plans to add CRM support to this library. Zoho provides an official Python binding for their CRM API at [https://www.zoho.com/crm/developer/docs/sdk/server-side/python-sdk.html](https://www.zoho.com/crm/developer/docs/sdk/server-side/python-sdk.html).
